doc/changes/latest.rst

```diff
@@ -8,7 +8,7 @@
 
 NEW
 ---
-
+- Volume plotting functions like :func:`~plotting.plot_img` now have an optional ``radiological`` parameter, defaulting to ``False``. If ``True``, this will invert the x-axis and ``L`` and ``R`` annotations to confirm to radiological conventional view. (:gh:`3172` by `Konrad Wagstyl`_).
```

Comment:
Since this is your first contribution to Nilearn, you will need to add your name as well as the link to your profile/website in `doc/changes/names.rst` for this to render well.

Reply:
name added